lib/factory_bot/definition_proxy.rb

```diff
@@ -176,6 +176,48 @@ def trait(name, &block)
       @definition.define_trait(Trait.new(name, &block))
     end
 
+    # Allows for creation of traits for enumerable values. The instance can be
+    # built with the following allowed syntax.
+    # ----
+    # Default Example:
+    #   factory :task do
+    #     traits_for_enum :status
+    #   end
+    #
+    # Arguments:
+    # * name_of_enum_type_attribute: +Symbol+
```

Comment:
I don't think we use this name anywhere else. I think we should stick to the argument names we use in the method signature itself.